distiller/quantization/range_linear.py

```diff
@@ -585,8 +614,10 @@ def __init__(self, wrapped_module, num_bits, mode=LinearQuantMode.SYMMETRIC, sta
         else:
             w_scale, w_zero_point = _get_quant_params_from_stats_dict(stats['output'], num_bits, mode)
 
-        self.register_buffer('w_scale', w_scale)
-        self.register_buffer('w_zero_point', w_zero_point)
+        device = next(wrapped_module.parameters()).device
```

Comment:
```suggestion
        device = wrapped_module.weight.device
```
You know it's an `Embedding` module with a `weight` parameter, no need to be too generic here.